runTests.py

```diff
@@ -97,10 +97,12 @@ def makeTest(name, j):
     """Run the make command for a given single test."""
     doCommand('make -j%d %s' % (j or 1, name))
 
-def runTest(name, run_all=False):
+def runTest(name, run_all=False, mpi=False):
     executable = mungeName(name).replace("/", os.sep)
     xml = mungeName(name).replace(winsfx, "")
     command = '%s --gtest_output="xml:%s.xml"' % (executable, xml)
+    if mpi and "mpi_" in name:
```

Comment:
This logic should be changed to just `if mpi:`. We want to make sure that when we specify MPI in the build and even if nothing uses MPI, it should still run.

Reply:
This is just saying _how_ to run it - they will all run with MPI compiled in, but some will be run with `mpirun`. Is there some reason we want to run non-mpi unit tests with `mpirun`? (also does that actually work?)

Reply:
To answer my 2nd question, it does still run with `mpirun` even if they aren't mpi tests.